add-bluecloth-gem

From: Bryan Larsen <bryan@larsen.st>

You may need to install the relevant ruby gem: either BlueCloth
(markdown) or RedCloth (textile).   Do this by adding

    gem "bluecloth"

to your Gemfile and then running


    $ bundle install
---
 Gemfile      |    1 +
 Gemfile.lock |    2 ++
 2 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/Gemfile b/Gemfile
index aec4203..81dbbb0 100644
--- a/Gemfile
+++ b/Gemfile
@@ -30,3 +30,4 @@ gem 'sqlite3'
 #   gem 'webrat'
 # end
 gem "hobo", ">= 1.3.0.pre27"
+gem "bluecloth"
diff --git a/Gemfile.lock b/Gemfile.lock
index b65a97a..ee27e42 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -29,6 +29,7 @@ GEM
       activesupport (= 3.0.4)
     activesupport (3.0.4)
     arel (2.0.9)
+    bluecloth (2.1.0)
     builder (2.1.2)
     dryml (1.3.0.pre27)
       actionpack (>= 3.0.0)
@@ -84,6 +85,7 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
+  bluecloth
   hobo (>= 1.3.0.pre27)
   rails (= 3.0.4)
   sqlite3
